packages/eslint-config-custom/index.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     'jsx-a11y/alt-text': 'off',
     '@next/next/no-img-element': 'off',
     '@next/next/no-html-link-for-pages': 'off',
+    'react/jsx-curly-brace-presence': ['error', { props: 'never', children: 'ignore' }],
```

Comment:
Random, but was testing whether this can mitigate VSCode inserting `{}` for every responsive prop—seems to work well, would like to keep.